package.json

```diff
@@ -48,18 +46,17 @@
     "test:rendering:update-screenshots": "./test/rendering/scripts/run-rendering-tests.sh '--update-snapshots'",
     "test:unit": "jest",
     "types:check": "tsc --noEmit",
-    "preversion": "yarn build",
-    "version": "conventional-changelog -p angular -i CHANGELOG.md -s && yarn spec && git add api-specifications CHANGELOG.md"
+    "preversion": "pnpm build",
+    "version": "conventional-changelog -p angular -i CHANGELOG.md -s && pnpm spec && git add api-specifications CHANGELOG.md"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": [
-      "bash -c 'yarn lint'",
-      "bash -c 'yarn types:check'"
+    "*.{js,jsx,ts,tsx, yaml}": [
```

Comment:
I think we only have `.yml` files

Reply:
It's meant to catch changes in `pnpm-lock.yaml`. Not ideal but since the pattern here was only using file ends I ended up doing like this